verticapy/tests_new/plotting/highcharts/machine_learning/test_highcharts_lof.py

```diff
@@ -0,0 +1,41 @@
+"""
+(c)  Copyright  [2018-2023]  OpenText  or one of its
+affiliates.  Licensed  under  the   Apache  License,
+Version 2.0 (the  "License"); You  may  not use this
+file except in compliance with the License.
+
+You may obtain a copy of the License at:
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless  required  by applicable  law or  agreed to in
+writing, software  distributed  under the  License is
+distributed on an  "AS IS" BASIS,  WITHOUT WARRANTIES
+OR CONDITIONS OF ANY KIND, either express or implied.
+See the  License for the specific  language governing
+permissions and limitations under the License.
+"""
+# Pytest
+import pytest
+
+
+# Verticapy
+from verticapy.tests_new.plotting.base_test_files import LOFPlot2D, LOFPlot3D
+
+
+# Testing variables
+COL_NAME_1 = "X"
+COL_NAME_2 = "Y"
+COL_NAME_3 = "Z"
```

Comment:
These variables look redundant. The LOFPlot2D and LOFPlot3D classes have class variables with the same names and use those in their functions.

Reply:
done